### DISCUSSION TITLE
NEXT.js@latest  net::ERR_ABORTED 405

### DISCUSSION THREAD

**Discussion by kysclient** [vercel/community#3138] [Help] [answered]
### Summary

If an api route that works well in the local environment is deployed to vercel, the **_handler cannot be accessed_** and a 405 error occurs... Please find the answer for what the reason is. I don't think there's any problem with the code, but I'll upload it.
Don't worry about the code, **_remember you can't access the handler_**. I wrote the path correctly and the file is under /pages/api.

### Steps to Reproduce

**here is my client code**

   const response = await fetch(endpoint, {
                method: 'POST',
                headers: {
                    'Content-Type': 'application/json',
                },
                signal: controller.signal,
                body,
            });

**and here is my server code**

export const config = {
    runtime: 'edge',
};

export default async function handler (req: Request): Promise<Response>  {
          // some code
}

**Comment by amyegan** [comment]
Hi @kysclient. If you share your public repo or a minimal reproduction, that will give us a better view of what is happening.

**Reply by kysclient** [reply]
you can see here that what's problem 

https://todaysmeet.vercel.app/explore/


Sorry to bother you, but please register as a member first.  
And try chatting and turn on developer tools to see my problem 
@amyegan

**Reply by kysclient** [reply]
@amyegan  hey..Amy,,. please help me..

**Reply by amyegan** [reply]
I need a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to be able to investigate what is going wrong

**Reply by kysclient** [reply]
@amyegan  no.... you don't need to some example..  even if I send a post request to a place that does not exist in /pages/api, the same 405 occurs. please tell me the cause of this

**Reply by kysclient** [reply]
![스크린샷 2023-07-13 오후 5 37 39](https://github.com/vercel/community/assets/85853598/cd63bf38-bb80-4877-86bc-eda0ebe9772d)


whats........wrong,,,,,,,

**Reply by HournNara** [reply]
@kysclient may I ask by any change you are using Prisma in your code?

**Reply by kysclient** [reply]
@HournNara  I dont use Prisma..

**Reply by HournNara** [reply]
@kysclient  I see, well in any case, you might want to check out vercel's log. I had exact same issue showing 405, and found out from the log that prisma was not initialize properly after the build. So you might want to start there.

**Reply by kysclient** [reply, marked as answer]
Sorry every body..I solved this issue.  the reason is why i have to build .next directory but  i built out directory

**Reply by sangonzal** [reply]
@kysclient I'm running into this same issue, could you expand on your solution? What do you mean by having to build .next directory?

**Comment by tariq-khan9** [comment]
@kysclient, same problem here. could you tell us how you resolved it. thank you
